examples/with-realm-web/next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  env: {
+    REALM_APP_ID: 'realm-example-bspbt',
+  },
+}
```

Comment:
Remove this file and use `.env` instead, the same way you had it before, rename the env var to be `NEXT_PUBLIC_REALM_APP_ID`, that way it will be available to the browser

Reply:
Done in 8d58d36